chapter_attention-mechanism/seq2seq-attention.md

```diff
@@ -88,12 +94,13 @@ out.shape, len(state), state[0].shape, len(state[1]), state[1][0].shape
 
 ## Training
 
-Again, we use the same training hyper-parameters as in
-:numref:`sec_seq2seq`. The training loss is similar to the seq2seq model, because the
+Similar to :numref:`sec_seq2seq_training`, we try a toy model by applying
+the same training hyper-parameters and the same training loss.
+As we can see from the result, since the
 sequences in the training dataset are relative short. The additional attention
```

Comment:
As we can see from the result, since the
sequences in the training dataset are relative short.


Incomplete sentence